build_projects/dotnet-host-build/PublishTargets.cs

```diff
@@ -230,6 +230,7 @@ private static bool CheckIfAllBuildsHavePublished()
             nameof(PublishTargets.PublishDotnetDebToolPackage),
             nameof(PublishTargets.PublishDebFilesToDebianRepo),
             nameof(PublishTargets.PublishCoreHostPackages),
+            nameof(PublishTargets.PublishSignedPackages),
```

Comment:
Hmm, would it be accurate to say `PublishManagedPackages`? A bit more exact to what's being published... although it misses out on the reason. Not sure.